src/routes/wallet/atomic-swap/tx-proposal.routes.js

```diff
@@ -70,4 +72,14 @@ txProposalRouter.post(
   getInputData,
 );
 
+txProposalRouter.get(
```

Comment:
Don't we need the `register` endpoint as well?

Reply:
The `register` endpoint requires fetching data from the atomic swap backend to ensure the proposal is valid.
This needs to be implemented together with the `fetch` endpoint that is already on PR 277.

Also added this information to the PR description.